### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by ksigei**
### In this milestone, I was able to:

- Initialise React app.

- The building blocks of my app is set as re-usable components. 

- Add Router and set two <Route>s and <Link>s for the app's navigation:

### CODE REVIEW

File: src/modules/Routes/Categories.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Categories() {
+  return (
+    <div>
+      <button type="button">Check status</button>
+    </div>
```
Comment (Cathella) [inline src/modules/Routes/Categories.jsx:7]: ✅ Should display only a button with text "Check status" 👍🏻

File: src/modules/Routes/NewBook.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+function NewBook() {
+  return (
+    <div>
+      <div className="container">
+        <h3>ADD NEW BOOK</h3>
+        <form>
+          <input placeholder="Book Title" />
+          <input placeholder="Book Author" />
+        </form>
+        <button type="button">Add book</button>
```
Comment (Cathella) [inline src/modules/Routes/NewBook.js]: ✅  Renders a form with inputs for the title and the author of the book 👍🏻 
✅  Has a button to submit 👍🏻

File: src/modules/Routes/BookList.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import NewBook from './NewBook';
+
+function BookList() {
+  return (
+    <div>
+      <Book />
+      <Book />
+      <Book />
+      <NewBook />
+    </div>
```
Comment (Cathella) [inline src/modules/Routes/BookList.js]: ✅  Renders a list of books using the `Book` component 👍🏻 
✅  Renders a form by using the NewBook component 👍🏻

File: src/modules/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+import {Link} from 'react-router-dom';
+import './scss/Header.scss';
+
+function Header() {
+  return (
+    <div>
+        <nav>
+            <Link className="title_link" to="/">Bookstore CMS</Link>
+            <Link className="link" to="/books">BOOKS</Link>
+            <Link className="link" to="/categories">CATEGORIES</Link>
```
Comment (Cathella) [inline src/modules/Header.jsx]: ✅ Has 2 `<Link>`s in the navbar which leads to the booklist and categories locations 👍🏻

File: src/App.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+import Categories from './modules/Routes/Categories';
+
+function App() {
+  return (
+    <div className="App">
+      <Header />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/books" element={<BookList />} />
+        <Route path="/categories" element={<Categories />} />
```
Comment (Cathella) [inline src/App.js]: ✅ Has 2 `<Route>`s, one for the BookList component and the other for the Categories component 👍🏻 

- [ ] [REQUIRED] Kindly make the BookList component as the default page as requested in the project requirements

File: package.json [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.2.0",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^18.1.0",
+    "react-browser-router": "^2.1.2",
+    "react-dom": "^18.1.0",
+    "react-router-dom": "^6.3.0",
```
Comment (Cathella) [inline package.json:12]: ✅ `react-router-dom` should be present in `package.json`. It will be needed to make the routes 👍🏻

File: package.json [hunk 7]
```diff
@@ -0,0 +1,58 @@
+{
+  "name": "bookstore_react",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.2.0",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^18.1.0",
+    "react-browser-router": "^2.1.2",
```
Comment (Cathella) [inline package.json:10]: ✅ The project should be initialized using create-react-app 👍🏻

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @ksigei 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻

✅ No linter errors.
✅ You used correct GitHub Flow.
✅ Descriptive pull request

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 7 above)
Comment (ksigei) [reply package.json:10]: Hey, i initialized the app with create-react-app

File: src/modules/Routes/Categories.jsx (hunk 1 above)
Comment (ksigei) [reply src/modules/Routes/Categories.jsx:7]: Hi, this is a button

File: src/modules/Routes/NewBook.js (hunk 2 above)
Comment (ksigei) [reply src/modules/Routes/NewBook.js:12]: Here is a form with all requirements

**Review by ahmetbozaci — APPROVED** [review]
# STATUS: APPROVED 🥳 🥳 🥳 



@ksigei  ✋🏼 
#### Your project is great. 🎉 I approve of it. There is nothing else to say other than... it's time to merge it. You can submit this project on your dashboard.

___ 

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏

aproved

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Pull request merged**